index_new.php

```diff
@@ -166,7 +166,9 @@
                 if(strlen($e->getMessage()) > 1) {
                     // Display page title if page is set
                     if($routes[$id] != null) {
-                        echo str_replace("[HEADER_TITLE]", $route->title, $layout->body_start);
+                    require($layout->sidebarModule);
+                    require($layout->topbarModule);
+                    echo str_replace("[HEADER_TITLE]", $route->title, $layout->body_start);
```

Comment:
Would be great if we can pull these three lines to a $layout class helper method like `$layout->renderBodyStart(string $page_title)` that can then internally sort out what to require and display

Reply:
Making note for future `index_new` overhaul, most other logic from `index` and `layout` needs replacement.